README.md

```diff
@@ -27,16 +27,52 @@ directory layout:
            |-- <command2>.py
 
 ```
+
+pycog3 also supports multi-level by using the `-` field separator in the command name. 
+For example, defining:
+ * `command1` - maps to `!command1`
+ * `level1-level2-command` - maps to `!level1-level2-command`
+ * `level1-...-...-levelN-command` - maps to `!level1-...-...-levelN-command`
+ 
+
+```
+<bundle_name>
+  |
+  |-- <bundle_name>
+       |-- __init__.py
+       |-- commands
+           |
+           |-- __init__.py
+           |-- <command1>.py
+           |-- <level1>
+                |
+                |-- __init__.py
+                |-- <level2>.py
```

Comment:
Can we have both a `<level2>.py` and `<level2>` directory, and show how commands could be loaded from each?

Reply:
Done - in fact `<level2>` was done but wrongly added `.py`. Added `<command>` in `level1` to show that's also possible.